analyticsdataserver/settings/base.py

```diff
@@ -297,6 +303,8 @@
 
 ENABLE_ADMIN_SITE = False
 
+LMS_BASE_URL = ''
```

Comment:
Nit: this setting isn't used too much, but is referenced https://github.com/edx/edx-analytics-data-api/blob/5c635010f15650670a46af1ca52c14ad706d8f8a/analytics_data_api/management/commands/generate_fake_course_data.py#L300. It makes sense to have a default to this, but perhaps `None` instead?

Reply:
This was added to base because the enterprise-data api endpoint needs it as well (for now, someday we'd like to get the enterprise association into the JWT, but for now we are making a call to LMS to determine)

Reply:
Gotcha! It totally makes sense to have that here. I'm assuming it would be actually set in the configuration repo.

Does it mess up the enterprise-data api endpoint by setting it to `None` in `base.py`? My one concern is that if this isn't set to `None`, the `AttributeError` in https://github.com/edx/edx-analytics-data-api/blob/5c635010f15650670a46af1ca52c14ad706d8f8a/analytics_data_api/management/commands/generate_fake_course_data.py#L302 won't be hit for other environments. We could certainly update that piece of code to check for an empty string too...

Reply:
Oh good catch! Will update it to `None`

Reply:
🍰